controlers/topic-controler.js

```diff
@@ -0,0 +1,13 @@
+const { fetchTopics, invalidPaths } = require("../models/topic-model");
+
+exports.getTopics = (req, res, next) => {
+  fetchTopics().then((results) => {
+    res.status(200).send({ topics: results });
+  });
```

Comment:
This is really good, but I am not a big fan of results being the name of this variable. What is results? At this stage we should have confidence that this is the array of topics and can name it exactly that as well just for further clarification. 

This should also have a .catch as well, just incase something unexpected happens